devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -196,7 +179,7 @@ public void setPreventnoverify(boolean preventNoVerify) {
         getPreventNoVerify().set(preventNoVerify);
     }
 
-    @Input
+    @Internal
     public ListProperty<String> getJvmArguments() {
```

Comment:
What is the difference between `@Input` and `@Internal` for options ? shouldn't we keep a `@Input` here?

Reply:
Inherently Grade considers all getters as inputs of some sort, and gives you errors if they are not annotated appropriately.

`@Input` (or `@InputFile`, ...) define properties as part of the task's `Task#inputs` which are used for up-to-date checking.  

We never want these tasks to be considered up-to-date - they should always run.

`@Internal` is a way to mark a getter to tell Gradle to ignore it for up-to-date checking.  Gradle also recently added this new `@UntrackedTask` annotation which serves the same role, but at the task/class level.  To be honest, I am not sure what happens though if you use `@UntrackedTask` and leave off the specific getter annotations.  I can try if you like.

But regardless, `@Input` is not appropriate here imo